core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -919,6 +922,8 @@ object KafkaConfig {
   val LogRollTimeJitterMillisDoc = "The maximum jitter to subtract from logRollTimeMillis (in milliseconds). If not set, the value in " + LogRollTimeJitterHoursProp + " is used"
   val LogRollTimeJitterHoursDoc = "The maximum jitter to subtract from logRollTimeMillis (in hours), secondary to " + LogRollTimeJitterMillisProp + " property"
 
+  val LiMinLogRollTimeMillisDoc = "The minimum interval (in milliseconds) between new segment rolling out."
```

Comment:
Nit: This is the minimum interval for _scheduled_ log rolls (i.e., time-based). Size-based rollovers can still be faster than this. (Also: "rolling over"; "rolling out" means leaving [usually in vehicles].)

So something like:  "The minimum interval (in milliseconds) before a log segment can be forced to roll over due to time-based scheduling. Segments may still roll faster due to size limits."  (That's a little long, but it's surprisingly tricky to come up with a short but precise statement...)

Reply:
Addressed